src/ServiceManagement/Common/Commands.ScenarioTest/Resources/ServiceManagement/Files/dSMSTest.cscfg

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  **********************************************************************************************
+
+  This file was generated by a tool from the project file: ServiceConfiguration.Cloud.cscfg
+
+  Changes to this file may cause incorrect behavior and will be lost if the file is regenerated.
+
+  **********************************************************************************************
+-->
+<ServiceConfiguration serviceName="AzureCloudService1" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceConfiguration" osFamily="5" osVersion="*" schemaVersion="2015-04.2.6">
+  <Role name="WorkerRole1">
+    <Instances count="1" />
+    <ConfigurationSettings>
+      <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="UseDevelopmentStorage=true" />
+      <Setting name="DummySetting" value="Foo" />
+    </ConfigurationSettings>
+    <Certificates>
+      <Certificate name="ThumbprintCert" thumbprint="{{PLACEHOLDER}}" thumbprintAlgorithm="sha1" />
+      <Certificate name="dSMSCert" sourceLocation="/genevacustomers/monitoring/prod/warmpath/test/auxweb/certificates/chained/portaldogfoodmdscert.pfx" />
```

Comment:
Is it the correct source location format? A sample source location would be like sourceLocation="https://uswest-dsms.dsms.core.windows.net/dsms/AzureTeam/ABC/certificates/chained/myPFXName.pfx" as shown here https://microsoft.sharepoint.com/teams/AzureSecurityCompliance/Security/_layouts/OneNote.aspx?id=%2Fteams%2FAzureSecurityCompliance%2FSecurity%2FSpecs%20and%20Design%20Docs%2FdSMS%2FOnboarding%2FdSMS%20Support&wd=target%28dSMS%20Onboarding%2FCertificates.one%7C4D38427D-2849-456A-8AED-B754A7AAF6A2%2FOK%20%5C%2F%20Express%20v2%7CB17594EC-657B-4B59-A4E3-F5E510E716C7%2F%29
onenote:https://microsoft.sharepoint.com/teams/AzureSecurityCompliance/Security/Specs%20and%20Design%20Docs/dSMS/Onboarding/dSMS%20Support/dSMS%20Onboarding/Certificates.one#OK%20/%20Express%20v2&section-id={4D38427D-2849-456A-8AED-B754A7AAF6A2}&page-id={B17594EC-657B-4B59-A4E3-F5E510E716C7}&end

Reply:
The relative sourceLocation works if you are using RDFE to perform deployments. It will automatically use the closest dSMS server location.
I tested this deployment with that format and it does work.